tests/cross_functional/system_test/test_nfs_export_in_out_cluster.py

```
@@ -0,0 +1,2046 @@
+                        log.info(f"  Mount details: {stdout.strip()}")
+                        break
+                    else:
+                        log.info(
+                            f"Attempt {attempt + 1}/{max_retries}: {mount_point} "
+                            f"not yet accessible..."
+                        )
+                        log.error(
+                            f"findmnt failed: retcode={retcode}, "
+                            f"stdout={stdout!r}, stderr={stderr!r}"
+                        )
+                        ls_retcode, ls_stdout, ls_stderr = con.exec_cmd(
```

Ruff (0.15.21)

[warning] Unpacked variable `ls_retcode` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)